README.md

```diff
@@ -42,6 +42,7 @@ For comprehensive docs, tutorials, and examples see our [documentation](https://
 
 ## Project status
 
+- (Dec 28, 2022) announcing the release of Kairos 1.4, which includes self-coordinated automated HA deployment via p2p and with [KubeVIP](https://kube-vip.io/). Check out the [docs](https://kairos.io/docs/installation/p2p/) for more information on this exciting new feature! Support for Ubuntu, Debian, and Fedora is also added in the Kairos 1.4 release. Expanding the number of supported operating systems currently available to pick from!
```

Comment:
Shall we create blog posts for major release announcements in the future?

Reply:
Yup agreed and makes sense. At least for impactful releases with changes that are worth mentioning, but let's make it opt-in or the process can become quickly cumbersome.